README.md

```diff
@@ -241,8 +245,22 @@ unityTransport.SetRelayServerData(new RelayServerData(allocation, "wss"));
 unityTransport.UseWebSockets = true;
 ```
 
+**Note** that if using Unity Transport Package 2.4.0, the constructor for RelayServerData(Allocation alloc, string connectionType) does not exist anymore. Hence use below as a example to set the WSS connection type for WebGL builds.
```

Comment:
I might be wrong, but isn't the breaking condition not 
* using Unity Transport Package 2.4.0

but instead about 
* using the Unity Multiplayer Services package instead of the Unity Relay
(**com.unity.services.multiplayer** instead of **com.unity.services.relay**)


?